platform/android/jni_utils.cpp

```diff
@@ -32,6 +32,56 @@
 
 #include "api/java_class_wrapper.h"
 
+jobject callable_to_jcallable(JNIEnv *p_env, const Variant *p_callable) {
+	ERR_FAIL_NULL_V(p_env, nullptr);
+	ERR_FAIL_NULL_V(p_callable, nullptr);
+	if (p_callable->get_type() != Variant::CALLABLE) {
+		return nullptr;
+	}
+
+	jclass bclass = p_env->FindClass("org/godotengine/godot/variant/Callable");
+	jmethodID ctor = p_env->GetMethodID(bclass, "<init>", "(J)V");
+	jobject jcallable = p_env->NewObject(bclass, ctor, reinterpret_cast<int64_t>(p_callable));
```

Comment:
@dsnopek Thanks for the suggestion! I've updated the PR, and that also fixes the issue @syntaxerror247 was seeing where the toast doesn't show again. @syntaxerror247 Can you test again and check if the callable is called and the toast shows?

> and having the Callable class memdelete() it up when it is destroyed.

This part is more complex to do because java objects don't get callbacks when they're destroyed, so we don't have a mechanism to know when to clean up. I've omitted it for now, which means we're allocating `Variant *` but not freeing, so looking for suggestions.

Reply:
@m4gr3d I tested again, now it works!

Reply:
> This part is more complex to do because java objects don't get callbacks when they're destroyed, so we don't have a mechanism to know when to clean up. I've omitted it for now, which means we're allocating Variant * but not freeing, so looking for suggestions.

I ended up using the [Object::finalize()](https://developer.android.com/reference/java/lang/Object#finalize()) method in order to detect when to deallocate the native memory. 
As marked in the documentation, that method has been deprecated; unfortunately its replacement, [`java.lang.ref.Cleaner`](https://developer.android.com/reference/java/lang/ref/Cleaner) is only supported on Android api 33 and higher, so we'll need to update that logic when our min api target catches up.

Reply:
The solution to this issue in the latest version of the PR looks good to me!